app/src/webview/java/org/mozilla/focus/webview/SystemWebView.java

```diff
@@ -223,6 +223,11 @@ public void loadData(String baseURL, String data, String mimeType, String encodi
         loadDataWithBaseURL(baseURL, data, mimeType, encoding, historyURL);
     }
 
+    @Override
+    public void getTelemetrySnapshots() {
+        // Do nothing, for GeckoView
```

Comment:
Let's throw something like IllegalStateException here so that we notice if something goes wrong and we start calling this for WebView.